### PR TITLE
update master from staging

### PR DESCRIPTION
**Pull request by zmalik**
`staging` branch has been soaked for a week now. This is usual process of updating master branch after a week of soaking.

Please do not squash the commits of this PR but we need to merge the commits to keep the history
Reviewers can check view the catalog in soak cluster under `kudo-staging` project https://github.com/mesosphere/konvoy-soak/tree/master/aws 

Only thing updated in current release is KUDO Cassandra metadata

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-KUTTL_VERSION=0.1.0
+KUTTL_VERSION=0.5.0
```
Comment (nfnt) [inline Makefile:1]: Let's use the latest version of kuttl, 0.6.1
```suggestion
KUTTL_VERSION=0.6.1
```

File: Makefile [hunk 2]
```diff
@@ -37,9 +36,17 @@ install-bin: bin/kind bin/kubectl-kuttl
...
+	git clone --depth 1 https://github.com/mesosphere/kubeaddons-tests.git --branch master --single-branch
+
 $(KUBECONFIG): install-bin
-	bin/kind create cluster --wait 10s --config=test/kind/$(KUBERNETES_VERSION).yaml
+	bin/kind create cluster --wait 10s --image=kindest/node:v$(KUBERNETES_VERSION)
 
 .PHONY: kind-test
-kind-test: create-kind-cluster
-# TODO restore once catalog is tested
+kind-test: kubeaddons-tests create-kind-cluster
```
Comment (nfnt) [inline Makefile:46]: Not yours, but dependency resolution might not work as expected here. If a kind cluster already exists, running `make kind-test` will run `create-kind-cluster` again, probably resulting in errors. If only the `kind-test` target is expected to be run regularly by this Makefile, a batch script would be a better approach.
If other targets are used a lot, we should have targets create files that indicate the current state.

**Review by nfnt — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (zmalik) [reply Makefile:1]: should be updated in `dev` branch and not in this PR

**Comment by zmalik** [comment]
@shaneutt who can help us with dispatch here?

**Comment by shaneutt** [comment]
cc @faiq

**Comment by shaneutt** [comment]
Checking in https://mesosphere.slack.com/archives/CQ7J6EWG3/p1598890578050300

**Pull request merged**